### ISSUE TITLE
`make test` 8 failing tests on M1

### ISSUE THREAD

**Issue by PaulMest** [brycedrennan/imaginAIry#133]
Just tried cloning the repo to make some small enhancements and ran into an issue running the tests after I got my environment set up. Any idea if this is a known issue? Or something unique to my environment?

## Repro
```
$ git clone git@github.com:brycedrennan/imaginAIry.git
$ cd imaginAIry
$ make init
$ make test
```
## Results
```
(imaginairy-3.10.6) ➜  ~/code/CleverPoint/open-source/imaginAIry git:(master) make test
Test session starts (platform: darwin, Python 3.10.6, pytest 7.2.0, pytest-sugar 0.9.6)
cachedir: .pytest_cache
Using --randomly-seed=2699774877
rootdir: /Users/paul/code/CleverPoint/open-source/imaginAIry, configfile: tox.ini
plugins: randomly-3.12.0, sugar-0.9.6, pylama-8.4.1
collecting ...

...

=================================================================== short test summary info ===================================================================
FAILED tests/test_enhancers.py::test_clip_masking - AssertionError: 659.180 is bigger than threshold 600
FAILED tests/test_clip_embedder.py::test_text_conditioning - AssertionError: assert '2b99000a0d1312c74d0dddaa9d8bab9e' in {'263e5ee7d2be087d816e094b80ffc546', '41818051d7c469fc57d0a940c9d24d82', 'b5f29fb26bceb60dcde...
FAILED tests/test_api.py::test_imagine[k_euler_a] - AssertionError: 58819.921 is bigger than threshold 10000
FAILED tests/test_api.py::test_img2img_beach_to_sunset[k_euler_a] - AssertionError: 8686.553 is bigger than threshold 2800
FAILED tests/test_api.py::test_img_to_img_fruit_2_gold[k_euler_a-0] - AssertionError: 32330.323 is bigger than threshold 10000
FAILED tests/test_api.py::test_img_to_img_fruit_2_gold[k_euler_a-0.05] - AssertionError: 44484.977 is bigger than threshold 10000
FAILED tests/test_api.py::test_img_to_img_from_url_cats[k_euler_a] - AssertionError: 27796.881 is bigger than threshold 14000
FAILED tests/test_api.py::test_img_to_img_fruit_2_gold[k_euler_a-0.2] - AssertionError: 45445.230 is bigger than threshold 10000

Results (934.08s):
      40 passed
       8 failed
         - tests/test_enhancers.py:29 test_clip_masking
         - tests/test_clip_embedder.py:14 test_text_conditioning
         - tests/test_api.py:15 test_imagine[k_euler_a]
         - tests/test_api.py:70 test_img2img_beach_to_sunset[k_euler_a]
         - tests/test_api.py:131 test_img_to_img_fruit_2_gold[k_euler_a-0]
         - tests/test_api.py:131 test_img_to_img_fruit_2_gold[k_euler_a-0.05]
         - tests/test_api.py:94 test_img_to_img_from_url_cats[k_euler_a]
         - tests/test_api.py:131 test_img_to_img_fruit_2_gold[k_euler_a-0.2]
       1 xfailed
       6 skipped
make: *** [test] Error 1
```

**Comment by brycedrennan** [comment]
Ah sorry I didn't get back to you quicker. That's probably normal. The test suite takes forever. I actually have a scrip that spins up a cluster of GPUS and distributes it among them.  But it's hacked together and costs money so I haven't put it in this repo.

**Comment by brycedrennan** [comment]
I haven't run tests on the m1 for a while so yeah I'd expect some similarity checks aren't working. I would manually look at the generated images and if they look good, adjust the test to have them pass.

**Comment by PaulMest** [comment]
## `test_clip_masking`
`test_clip_masking` is fine with a modified threshold.

## `test_text_conditioning`
`test_text_conditioning` -- not sure what this one is actually testing. Should I just add the missing hash? (`2b99000a0d1312c74d0dddaa9d8bab9e`)

## `euler`
All of the euler ones show a visually different image
```
$ pytest -k "euler"
```

Examples...
| Expected  | Actual |
| ------------- | ------------- |
| ![test_imagine k_euler_a _](https://user-images.githubusercontent.com/1721104/206894088-4f853381-a132-424f-a42f-662a89d935cd.png)  | ![test_imagine k_euler_a _](https://user-images.githubusercontent.com/1721104/206894119-8455b7a1-9f10-46ff-8bae-8088ec9a2db5.png)  |
| ![test_img_to_img_from_url_cats k_euler_a _ png_expected](https://user-images.githubusercontent.com/1721104/206894183-6e5cad9e-5446-4b38-9d68-00b0cfca754f.png)  | ![test_img_to_img_from_url_cats k_euler_a _](https://user-images.githubusercontent.com/1721104/206894189-8f7b0544-4250-403a-ab05-3ef3770ff961.png)  |
| ![test_img_to_img_fruit_2_gold k_euler_a-0 05 _ png_expected](https://user-images.githubusercontent.com/1721104/206894271-5b6f5472-4702-49ee-aa8b-267274a21976.png)  | ![test_img_to_img_fruit_2_gold k_euler_a-0 05 _](https://user-images.githubusercontent.com/1721104/206894275-7d065d45-457a-42a7-a5fc-e4f33c70cf89.png) |

**Comment by brycedrennan** [comment]
I ran `pytest -k euler` and the images don't match for me either.  My output images don't even match your new ones though. Not sure why euler ones are unstable.  

If I had time I'd investigate the source of differences step by step but given the images still look acceptable I'd probably just change test thresholds to accept these differences.

**Comment by brycedrennan** [comment]
![test_img_to_img_from_url_cats k_euler_a _](https://user-images.githubusercontent.com/1217531/206916839-7f49e4e3-558e-457b-af61-a0309d88911a.png)
![test_imagine k_euler_a _](https://user-images.githubusercontent.com/1217531/206916853-5784ba2f-762a-4310-941c-fdd37657a883.png)
![test_img_to_img_fruit_2_gold k_euler_a-0 _](https://user-images.githubusercontent.com/1217531/206916870-3c4239c1-4f59-4874-902e-b49cb723bc42.png)

**Comment by brycedrennan** [comment]
@PaulMest okay if we close this?

**Comment by PaulMest** [comment]
I haven't submitted a fix for these yet, but it's your repo. You can run it however you'd like. :)

**Comment by brycedrennan** [comment]
good point

**Comment by brycedrennan** [comment]
Fixed in 7.3.0
